### PR TITLE
implemented the hero

### PR DESCRIPTION
**Pull request by hola-official**
@DevBigEazi #6  #muha/hero

### CODE REVIEW

File: src/sections/Hero.jsx [hunk 1]
```diff
@@ -1,5 +1,40 @@
...
+          <img
+            className="sm:hidden w-full h-auto object-cover"
+            src={images.heroMobile}
+            alt="hero"
+          />
+        </div>
+      </div>
+    </section>
+  );
 };
```
Comment (DevBigEazi) [inline src/sections/Hero.jsx:39]: review-1

The padding are too much. Stick with the designs

Comment (DevBigEazi) [reply src/sections/Hero.jsx:39]: review-2

change the flex direction from 678px screen downward.

**Review by DevBigEazi — CHANGES_REQUESTED** [review]

**Comment by DevBigEazi** [comment]
Make sure the form is submiiting data to google sheet

File: src/sections/Hero.jsx [hunk 2]
```diff
@@ -1,5 +1,40 @@
...
+          </h1>
+          <div className="space-y-2">
+            <p className="text-base sm:text-lg lg:text-xl font-normal">
+              Corem ipsum dolor sit amet, consectetur.
+            </p>
+            <p className="text-base sm:text-lg lg:text-xl font-normal">
+              Corem ipsum dolor sit amet, consectetur.
+            </p>
+          </div>
+          <button className="self-start mt-2 sm:mt-4 h-11 max-sm:w-full text-center border border-primary transition-all duration-500 hover:bg-whitesmoke rounded-lg text-primary text-base max-sm:text-center font-montserrat font-normal items-center py-3 px-7 ">
```
[the remark this answers is not in the document]
Comment (hola-official) [reply src/sections/Hero.jsx]: done sir

File: src/sections/Hero.jsx [hunk 3]
```diff
@@ -1,5 +1,40 @@
...
+            Join Our Community
```
[the remark this answers is not in the document]
Comment (hola-official) [reply src/sections/Hero.jsx]: resolved sir

File: src/sections/Hero.jsx (hunk 1 above)
Comment (hola-official) [reply src/sections/Hero.jsx:151]: done sir

**Comment by hola-official** [comment]
@DevBigEazi all the issue have being resolved sir

File: src/components/Modal.jsx [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+  if (!isOpen && !isClosing) return null;
+
+  return (
+    <div
+      className={`fixed inset-0 bg-black/10 backdrop-blur-sm z-30 transition-opacity duration-400 ${
+        isClosing ? "opacity-0" : "animate-slide-in-right opacity-100"
+      }`}
+    >
+      <div
+        className={`fixed inset-0 bg-[#0f0140] z-50 flex items-center max-w-[978px] max-sm:mt-14 sm:h-[493px] m-auto justify-center p-4 transition-transform duration-400 ${
```
Comment (DevBigEazi) [inline src/components/Modal.jsx:39]: leverage the color configuration in tailwind.config.js

File: src/components/Modal.jsx [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+        }`}
+      >
+        <button
+          onClick={handleClose}
+          className="absolute top-4 right-4 text-white text-4xl float-right"
+          aria-label="Close modal"
+        >
+          &times;
+        </button>
+        <div className="relative w-full max-w-md bg-[#0f0140] rounded-lg shadow-lg">
```
Comment (DevBigEazi) [inline src/components/Modal.jsx:50]: leverage the color configuration in tailwind.config.js

File: src/components/Modal.jsx [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+                  className="block text-sm sm:text-base mb-2"
+                >
+                  Email
+                </label>
+                <input
+                  type="email"
+                  id="email"
+                  name="email"
+                  placeholder="johndoe@gmail.com"
+                  className="w-full p-3 bg-[#1D143E] border border-[#FFFFFF80] rounded-lg text-white placeholder-gray-400"
```
Comment (DevBigEazi) [inline src/components/Modal.jsx:87]: leverage the color configuration in tailwind.config.js

File: src/components/Modal.jsx [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+                  By continuing, you agree to the JaraFi Platform Terms &
+                  conditions, Rewards Policy and Privacy Policy.
+                </label>
+              </div>
+              {submitError && (
+                <p className="text-red-500 text-sm">{submitError}</p>
+              )}
+              <button
+                type="submit"
+                className="w-full bg-[#F9F6F3] text-[#0F0140] py-3 rounded-lg font-normal hover:bg-gray-100 transition duration-300 disabled:opacity-50 disabled:cursor-not-allowed"
```
Comment (DevBigEazi) [inline src/components/Modal.jsx:109]: leverage the color configuration in tailwind.config.js

File: src/components/Modal.jsx [hunk 8]
```diff
@@ -0,0 +1,122 @@
+import React, { useEffect, useState } from "react";
+
+const Modal = ({ isOpen, onClose, onSubmit, isSubmitting, submitError }) => {
+  const [isAgreed, setIsAgreed] = useState(false);
+  const [isClosing, setIsClosing] = useState(false);
+
```
Comment (DevBigEazi) [inline src/components/Modal.jsx:6]: Add a comment to specify what this useEffect does....

File: src/sections/Hero.jsx [hunk 9]
```diff
@@ -1,5 +1,152 @@
...
+  const [isSubmitting, setIsSubmitting] = useState(false);
+  const [submitError, setSubmitError] = useState(null);
+  const [success, setSuccess] = useState("");
+  const [existingEmails, setExistingEmails] = useState([]);
+  const sheetdbapi = import.meta.env.VITE_SHEET_DB;
+
+  const handleOpenModal = () => {
+    window.document.body.classList.add("overflow-hidden");
+    setIsModalOpen(true);
+  };
```
Comment (DevBigEazi) [inline src/sections/Hero.jsx:17]: add a sapce before the next function

File: src/sections/Hero.jsx [hunk 10]
```diff
@@ -1,5 +1,152 @@
...
+  const handleCloseModal = () => {
+    window.document.body.classList.remove("overflow-hidden");
+
+    setIsModalOpen(false);
+    setSubmitError(null);
+  };
+
+  useEffect(() => {
```
Comment (DevBigEazi) [inline src/sections/Hero.jsx:25]: Add a comment to specify what this useEffect does

**Review by DevBigEazi — CHANGES_REQUESTED** [review]
Remove the codes you commented out

**Comment by DevBigEazi** [comment]
review-3

let the section paddingLeft be the same as the navbar

File: src/components/Modal.jsx (hunk 4 above)
Comment (hola-official) [reply src/components/Modal.jsx:39]: done

**Comment by hola-official** [comment]
> review-3
> let the section paddingLeft be the same as the navbar

done sir

File: src/components/Modal.jsx (hunk 7 above)
Comment (hola-official) [reply src/components/Modal.jsx]: have fix it sir

**Comment by DevBigEazi** [comment]
kindly resolve the conflict @hola-official

**Comment by DevBigEazi** [comment]
review-4
review-5
Reduce the spaces in between 1024px screen and 768px screen

**Comment by hola-official** [comment]
@DevBigEazi conflicts resolved sir

**Pull request merged**